### ISSUE TITLE
Vagrant up doesn't work on Windows

### ISSUE THREAD

**Issue by manthey** [girder/girder#2027]
When `vagrant up` is used on Windows (and possibly on linux when the girder repository is on certain file systems), it fails.  The current directory is mounted by Vagrant to /home/vagrant/girder, and node attempts to create symlinks within this directory.  However, the ability to make symlinks is not exposed through the vboxsf, so this fails.

VirtualBox says the ability to create symlinks in a mounted directory is a security issue (https://www.virtualbox.org/ticket/10085).  It looks like a won't fix (or at least not any time soon), though there are various work-arounds.

One solution would be to *not* install in the mounted local directory, but rather copy the current directory to some other path, and use ansible_local from there (`/home/vagrant/not_girder`).  This has the drawback that it wouldn't use the current girder code for the vagrant, but the virtue it doesn't spatter the local file system with node modules.

**Comment by msmolens** [comment]
An upstream solution for this would be welcome, especially one that still allows editing Girder code directly on the host. This would make it easier to develop core Girder when only a Windows machine is available, or for participants in hackfests who might only have a Windows machine.

**Comment by manthey** [comment]
At least as a workaround, one could bind mount a local directory to the girder/node_modules directory in the VirtualBox.  The build then works (since it is only npm that creates symlinks to npm binaries).

I know there are better and more official ways to do this, but, crudely, adding this near the end of devops/ansible/roles/girder/tasks/girder.yml works:
```
- name: Bind mount node_modules
  shell: |
    mkdir -p /home/vagrant/girder/node_modules
    mkdir -p /home/vagrant/girder_node_modules
    sudo mount --bind /home/vagrant/girder_node_modules /home/vagrant/girder/node_modules
```
If we want to use that approach, we'd want the bind mount to automatically be done when `vagrant up` is done on an already provisioned VM, too.

**Comment by manthey** [comment]
Resolved in PR #2041.
